fe/src/project/AspNetCore/areas/News/components/NewsCatalog/NewsCatalog.scss

```diff
@@ -0,0 +1,90 @@
+.news-catalog {
+
+    .image img {
+        border-radius: 0.5rem;
```

Comment:
**suggestion:** Please use bulma helpers for it and remove it completely: https://bulma.io/documentation/helpers/